flatpak/io.github.qtox.qTox.json

```diff
@@ -2,14 +2,14 @@
     "app-id": "io.github.qtox.qTox",
     "runtime": "org.kde.Platform",
     "sdk": "org.kde.Sdk",
-    "runtime-version": "6.7",
+    "runtime-version": "5.15-24.08",
```

Comment:
you most certainly dont want that.

Reply:
Done. I got this from https://github.com/flathub/io.github.qtox.qTox/blob/master/io.github.qtox.qTox.json but yeah, makes sense.

Reply:
yes, I changed that so that it builds at all.